CMS-Enterprise/sbom-harbor-ui
# build(upload-ui): add dependencies

## Summary

This Pull Request adds new dependencies to support the new Upload V2 UI.

### Added

- The `react-animate-height` library (version 3.1.1) has been added as a dependency. This library provides a React component to animate an element's height from `0` to its contents height.
- The `react-dropzone` library (version 14.2.3) has been added as a dependency. This library provides a React component that supports file dragging and dropping.
- The `jest-date-mock` library (version 1.0.8) has been added as a development dependency. This library allows developers to mock JavaScript Date objects in Jest tests.

### Changed

- n/a

### Deprecated

- n/a

### Removed

- n/a

### Fixed

- n/a

## How to test

- After pulling changes, run `yarn` to install the new dependencies.
- Ensure all tests still pass with `yarn test`
